app/assets/stylesheets/flex_layout.css.scss

```diff
@@ -125,6 +126,7 @@ body {
     }
   }
    height: 40px;
+   min-height: 40px;
```

Comment:
This also feels wrong, but `height` alone is ignored:
<img width="289" alt="chrome_2016-04-08_12-00-15" src="https://cloud.githubusercontent.com/assets/133987/14391439/9d9d6b02-fd81-11e5-878a-30cf4eae4629.png">